src/mono/wasm/Wasm.Build.Tests/BuildTestBase.cs

```diff
@@ -676,11 +676,11 @@ static void AssertRuntimePackPath(string buildOutput, string targetFramework)
             var filesToExist = new List<string>()
             {
                 mainJS,
-                "dotnet.native.wasm",
+                "_framework/dotnet.native.wasm",
                 "_framework/blazor.boot.json",
```

Comment:
In one of the other files in this PR blazor.boot.json *isn't* in _framework, is this correct?

Reply:
`blazor.boot.json` is in `_framework`
but this is testing it from windows/linux file system perspective.

Reply:
The blazor.boot.json is always in the _framework dir, but the other WBT part for blazor tests has a _framework as "base dir" where it looks for files.